### ISSUE TITLE
Possibility to clear call history

### ISSUE THREAD

**Issue by berlinbikerx** [64characters/Telephone#420]
It should be possible to clear the call history of an account (eg. via a context menu)

**Comment by eofster** [comment]
Yep, there should be a menu item for that. Currently it is possible to delete the entries one by one by pressing the Return key.

**Comment by berlinbikerx** [comment]
Actually it's the delete key - the return key redials the entry :-)

**Comment by eofster** [comment]
Oh, right!

**Comment by Josecaballero322** [comment]
AA AA AA B I " <>(-)
